SnapMD.VirtualCare.ApiModels/ScheduledConsultationResult.cs

```diff
@@ -13,5 +14,10 @@ public class ScheduledConsultationResult
         /// The ID of the newly-created consultation.
         /// </summary>
         public int ConsultationId { get; set; }
+
+        /// <summary>
+        /// The UID of the newly-created consultation request.
```

Comment:
@jay-lee00 can you call it either "Guid" or "UniqueId" then? Also, why are we calling it "ConsultationRequest" instead of "AppointmentRequest"